openQA.spec

```diff
@@ -481,4 +489,8 @@ fi
 %files local-db
 %{_unitdir}/openqa-setup-db.service
 
+%files bootstrap
+%{_datadir}/openqa/script/openqa-bootstrap
+%{_datadir}/openqa/script/openqa-bootstrap-container
+
```

Comment:
How about moving these to /usr/bin on install as they already have an openqa-prefix?

Reply:
Having them in this special dir makes it harder to accidentally execute this scripts.